plugin/completion.py

```diff
@@ -69,7 +69,7 @@ def format_completion(
         annotation = lsp_label_description or lsp_detail
     elif lsp_label.startswith(lsp_filter_text):
         trigger = lsp_label
-        annotation = lsp_detail
+        annotation = lsp_label_description or lsp_detail
```

Comment:
One datapoint from Pyright.

Before:

![Screenshot 2023-03-04 at 22 44 01](https://user-images.githubusercontent.com/153197/222930091-997d438e-21f2-4d2c-b7b7-253ea4103b97.png)

After:

![Screenshot 2023-03-04 at 22 39 32](https://user-images.githubusercontent.com/153197/222930095-97e97448-a220-471b-b41c-5d1f467d143d.png)

I would say that it makes things better in this case too.